code/modules/emotes/definitions/audible.dm

```diff
@@ -3,10 +3,40 @@
 	emote_message_3p = "USER burps."
 	message_type = AUDIBLE_MESSAGE
 	var/emote_sound
+//Urist add
+	var/gendered
+	var/male_sound //Give these strings for use through the sound.dm sound lists.
+	var/female_sound
+
+//Audible emotes with sound.
+client/
+	var/sound_cooldown = 0 //Can we do an emote with sound?
 
 /decl/emote/audible/do_extra(var/atom/user)
+	var/client/C = user.get_client()
+	if(isnull(C))
+		return //Dunno how this could happen, let's just be safe.
+	if(C.sound_cooldown == TRUE)
+		to_chat(user, "<span class = notice>Quiet down, you already made a noisy emote!</span")
+		return
 	if(emote_sound)
-		playsound(user.loc, emote_sound, 50, 0)
+		if(gendered)
+			var/gen = C.mob.get_gender()
+			switch(gen)
+				if("male")
+					gen = "male"
+				if("neuter")
+					gen = "male" //Functionality for neuterscreams is possible, but it'd require sounds.
+				if("female")
+					gen = "female"
+			playsound(user.loc, "[gen][key]" , 50, 0)
+		else
+			playsound(user.loc, emote_sound, 50, 0)
```

Comment:
The whole block from L23 could be rewritten more cleanly with an assoc list. Something like: 
`var/list/soundmap = list("female"="female"); var/soundkey=soundmap[gen]; if(!soundkey) soundkey = male` (or go into the non-gendered block, your design call).

Reply:
Handled.